CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [1.34](https://github.com/brave/brave-ios/releases/tag/v1.34)
+
+- Added support for initial set of Brave Widgets. ([#4329](https://github.com/brave/brave-ios/issues/4329))
+- Implemented "Login & Passwords" feature for managing saved credentials. ([#4112](https://github.com/brave/brave-ios/issues/4112))
+- [Security] Fixed reader mode issue as reported on HackerOne by nishimunea. ([#4772](https://github.com/brave/brave-ios/issues/4772))
+- [Security] Fixed message token handling as reported on HackerOne by nishimunea. ([#4775](https://github.com/brave/brave-ios/issues/4775))
+- [Security] Added "no-referrer" for content for all internal pages as reported on HackerOne by nishimunea. ([#4784](https://github.com/brave/brave-ios/issues/4784))
+- Removed Canvas Fingerprinting protection. ([#4522](https://github.com/brave/brave-ios/issues/4522))
+- Fixed Wallet page indicator width on portfolio screen. ([#4586](https://github.com/brave/brave-ios/issues/4586))
+
+## [1.33](https://github.com/brave/brave-ios/releases/tag/v1.33)
+
+- Added first steps of Brave Wallet (Dapps and EVM support in future releases).([#4837](https://github.com/brave/brave-ios/issues/4837))
+- Implemented drag-and-drop option to rearrange favorites on New Tab Page. ([#3713](https://github.com/brave/brave-ios/issues/3713))
+- Improved auto-fill of saved credentials on websites. ([#4113](https://github.com/brave/brave-ios/issues/4113))
+- Implemented time-limited sync QR codes. ([#4534](https://github.com/brave/brave-ios/issues/4534))
+- Fixed last active tab not being focused when closing and relaunching the browser. ([#736](https://github.com/brave/brave-ios/issues/736))
+- Fixed search feature on PDF file loaded on a webpage. ([#3312](https://github.com/brave/brave-ios/issues/3312))
```

Comment:
This one sounds a bit confusing. Maybe we could use something similar to the following:

```
Fixed searching opened PDF's within Brave not working in certain cases.
```